cypress/fixtures/README.md

```diff
@@ -2,6 +2,24 @@
 
 Fixtures are designed to load a fixed set of data located in a file.
 
-## Required data need to be updated before running the feature files
-### Advertising: 
- - APPBUNDLEID - It contains the corresponding app ID and should be updated in `configModule/testData/modules/advertising.json` for content validation.
+## Override concepts:
+-  If a key is present in FCS but not in the ConfigModule, the FCS key will be used without any overrides.
+-  If a key is present in the ConfigModule but not in FCS, The key from the ConfigModule is copied over and used.
+-  If a key is present in both FCS and the ConfigModule, the ConfigModule's key will override or merge with precedence given to the ConfigModule.
+
+## Directory strucute:
+Within the `cypress/fixtures` folder we have the following sub-folders:
+- fireboltCalls: Contains FireboltCall keys which is used to make firebolt calls to the 1st party app or 3rd party app.
+- fireboltMocks: Contains FireboltMock keys which is used to override the default response or set the responses.
+- modules: Contains json files in the format of <moduleName.json> where required content/parameters are present.
```

Comment:
![image](https://github.com/rdkcentral/firebolt-certification-suite/assets/148241508/8f2c986a-c629-42c7-8f88-fecefcaca611)
below 2 folders are missing

Reply:
Included these 2 folders in readme.